src/ledgers/five-bells-ledger.js

```diff
@@ -183,7 +204,11 @@ class FiveBellsLedger extends EventEmitter2 {
     return null
   }
 
-  * fulfillCondition (transferID, conditionFulfillment) {
+  fulfillCondition (transferID, conditionFulfillment) {
+    return co.wrap(this._fulfillCondition).call(this, transferID, conditionFulfillment)
```

Comment:
what's the point of using `co.wrap` for this and `getConnectors`? why not just change those functions to return promises?